pkgs/data/fonts/nerd-fonts/update.py

```diff
@@ -0,0 +1,44 @@
+#! /usr/bin/env nix-shell
+#! nix-shell -p python3 -i python3
+import os
+import urllib.request as ureq
+import json
+import subprocess
+
+RELEASE_INFO_URL = "https://api.github.com/repos/ryanoasis/nerd-fonts/releases/latest"
+FONTS_INFO_URL_TEMPLATE = "https://raw.githubusercontent.com/ryanoasis/nerd-fonts/refs/tags/{}/bin/scripts/lib/fonts.json"
+SHA256_URL_TEMPLATE = "https://github.com/ryanoasis/nerd-fonts/releases/download/{}/SHA-256.txt"
+
+RELEASE_INFO_FILENAME = "release.json"
+FONTS_INFO_FILENAME = "fonts.json"
+HASHES_FILENAME = "hashes.json"
+
+def converthash(s):
+    return subprocess.run([
+        "nix-hash",
+        "--to-sri",
+        "--type", "sha256",
+        s
+    ], capture_output=True, text=True).stdout.strip()
```

Comment:
I think Nix would go along just fine without this conversion, if you'd use the `sha256 = "<original-sha256-from-upstream>"` argument in the `fetchurl` call.